docroot/sites/all/themes/custom/wsdot_classic/css/styles.css

```diff
@@ -112,35 +113,42 @@ input.searchButton {height: 23px;margin: 0;font-family: "trebuchet ms",helvetica
 .corners-top {width:955px;margin:5px auto 0px auto;padding:5px;background:#fff url(../images/back/corners.png) no-repeat -965px top;height:auto;}
 .corners-bottom {width:955px;margin:0em auto;padding:5px;background:#fff url(../images/back/corners.png) no-repeat left bottom;height:5px;}
 
-/*** Left navigation column ***/
-.greyBg{background:#fff url(../images/back/gradients_bg.png) repeat-x left 0px;}
-.blueBg{background:#fff url(../images/back/gradients_bg.png) repeat-x left 0px;}
-.leftnav{display:table-cell;width:168px;}
-.leftnav ul{margin:0em 0em 0em .5em;padding-left:0em;font-size:1.2em;}
-.leftnav ul li ul{font-size:1em;}
+/* Left navigation column */
+.greyBg{background:#fff url(https://www.wsdot.wa.gov/media/images/back/gradients_bg.png) repeat-x left 0px;}
+.blueBg{background:#fff url(https://www.wsdot.wa.gov/media/images/back/gradients_bg.png) repeat-x left 0px;}
```

Comment:
The above two URLs need to stay as relative links to the `gradients_bg.png` file so they reference the image locally in Drupal rather than the static HTML server.